### DISCUSSION TITLE
Help with understanding extension sample projects

### DISCUSSION THREAD

**Discussion by Brackets-Coder** [TurboWarp/extensions#1680] [Q&A]
I've looked around for files that show which .sb3 sample projects correspond to what extension, but can't find any. When I'm submitting a PR for an extension, what should I do to add a sample project?

**Comment by CST1229** [comment]
I think the extension the sample project corresponds to is just automatically detected based on which extensions the project uses.

**Reply by Brackets-Coder** [reply]
[Looking at a previous discussion](https://github.com/TurboWarp/extensions/discussions/915#discussioncomment-6809098), it seems like there are manual options as well, but that may be true. Should I wait until my extension gets merged before releasing a sample project so that it pulls from extensions.turbowarp.org?
